src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -580,6 +441,10 @@ public function setAttribute($key, $value)
             $value = $this->fromDateTime($value);
         }
 
+        if ($this->isCustomCastable($key) && ! is_null($value)) {
```

Comment:
Are we sure we want to skip custom casts for null values? Maybe a custom caster needs to represent nulls or "no value" in a different manner? It's a "custom_ caster, after all?

Reply:
I agree. I will fix it as soon as I am at the computer.

Reply:
I fixed the condition by removing ignoring NULL values.